src/main/webapp/js/userMapData.json

```diff
@@ -456,6 +479,7 @@
 ["Dalton State College","USA"],
 ["Dartmouth College","USA"],
 ["Davis Senior High School","USA"],
+["Dawson high school","USA"],
```

Comment:
Capitalise this to keep it consistent with the other entries: `"Dawson High School"`